pyaedt/edb.py

```diff
@@ -1363,17 +1372,18 @@ def import_gds_file(
             have the same name. Only the extension differs.
         tech_file : str, optional
             Technology file. It uses Helic to convert tech file to xml and then imports the gds. Works on Linux only.
```

Comment:
```suggestion
            Technology file. This method uses Helic to convert the technology
            file to an XML file and then imports the GDS file. This method works on Linux only.
```
Does a technology file have a TECH extension (implied by the original wording of "convert tech file)?
Is a technology file a GDS files? I'm also confused about how we get from an XML file to importing a GDS file.

Reply:
can be multiple formats. that's why I didn't add the specific extension. It depends on the third party vendor